project/Build.scala

```diff
@@ -269,6 +270,27 @@ object ScalikeJDBCProjects extends Build {
     )
   ) dependsOn(scalikejdbcCore)
 
+  // scalikejdbc-streams
+  lazy val scalikejdbcStreams = Project(
+    id = "streams",
+    base = file("scalikejdbc-streams"),
+    settings = baseSettings ++ mimaSettings ++ Seq(
+      name := "scalikejdbc-streams",
+      libraryDependencies ++= {
+        Seq(
+          "org.reactivestreams" %  "reactive-streams"     % _reactiveStreamsVersion % "compile",
+          "org.slf4j"           %  "slf4j-api"            % _slf4jApiVersion        % "compile",
+          "ch.qos.logback"      %  "logback-classic"      % _logbackVersion         % "test",
+          "org.reactivestreams" %  "reactive-streams-tck" % _reactiveStreamsVersion % "test"
+        ) ++ scalaTestDependenciesInTestScope(scalatestVersion.value) ++ jdbcDriverDependenciesInTestScope
+      },
+      unmanagedSourceDirectories in Compile <+= (scalaVersion, sourceDirectory in Compile){(v, dir) =>
```

Comment:
`<+=` is deprecated since sbt 0.13.13

http://www.scala-sbt.org/0.13/docs/Migrating-from-sbt-012x.html